dockerfiles/slurm/slurm.submit

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#SBATCH -A IPLANT-DEV
+#SBATCH -J slurmtest
+#SBATCH -o slurmtest.out
+#SBATCH -e slurmtest.err
+#SBATCH -t 0:30:00
+#SBATCH -p debug
+#SBATCH -n 1
+
+date
+hostname
+sleep 5
+date
```

Comment:
What's the purpose of this file?

Reply:
the script used to submit a sample job. Prints time, hostname, sleeps, then prints date again. I guess it could just print hostname or something and avoid sleeping perhaps? (quite sure I copied it from some blog/doc)

Reply:
I guess we should cut the sleep out at least.

Reply:
Agreed, good catch! Will update in the morning.